torch/nn/grad.py

```diff
@@ -1,307 +1,16 @@
 """Gradient interface"""
```

Comment:
Maybe file an issue reminding ourselves to fully delete the file in 1.5, and link it from here?

Reply:
Well, having a functional interface for gradients would be nice I think. In particular for people implementing custom `Function`s that are similar to the original one.
But it should point to the functions that actually run during the regular backward.